LoopKitTests/GuardrailTests.swift

```diff
@@ -11,6 +11,86 @@ import HealthKit
 @testable import LoopKit
 
 class GuardrailTests: XCTestCase {
+    let correctionRangeSchedule120 = GlucoseRangeSchedule(unit: .milligramsPerDeciliter, dailyItems: [RepeatingScheduleValue(startTime: 0, value: DoubleRange(120...130))])
```

Comment:
consider adding a unit test that expected values in mmol/L

Reply:
I've removed the `unit` parameter and now all the functions deal in `HKQuantity`, so hopefully that obviates the need to test different units...?

Reply:
I still this is may be useful to add such a test to detect regression. But I leave it to you to decide.

Reply:
By that I mean it should be general practice to include tests for both types of glucose units. It is probably enough to just update one of the existing test to `mmol/L`.

Reply:
I'll add one test, just as a sanity check, but since all the math and comparisons are in terms of `HKQuantity`, there should not need to be complete coverage on that, IMO.